samples/bluetooth/iso_receive/prj.conf

```diff
@@ -2,3 +2,5 @@ CONFIG_BT=y
 CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_DEVICE_NAME="Test ISO Receive"
 CONFIG_BT_ISO_SYNC_RECEIVER=y
+
+CONFIG_MAIN_STACK_SIZE=2096
```

Comment:
This should be architecture/board specific and repository wide change if the increase is likely for all samples/feature used on nRF53 Series

Reply:
Removed reference to nRF53; I don't see this as being specific to that board nor the nRF family.

Reply:
is the iso_receive crashing on nrf52dk_nrf52832 and nrf52840dk_nrf52840, we do not want to use RAM foot print for architectures that do not use more program stack. Have you run with CONFIG_THREAD_RUNTIME_STATS=y? What are the values for stack usage?

Reply:
I can try to run some statistics tomorrow :) The `struct thread_analyzer_info` doesn't seem to have a `max stack size used`, so will need to figure out where it maxes out.

Reply:
It'll take a few more days to test this :)

Reply:
For debugging purposes I had `CONFIG_LOG_FUNC_NAME_PREFIX_ERR` enabled, which is what caused the stack overflow.

Reply:
@cvinayak The projects builds and runs with both nRF52480 and nRF5340. Should we close the PR, or does it make sense to still investigate the stack usage? It's seeming quite near the limits it seems.

Reply:
You can close the PR.